cassandra/src/test/java/se/tre/freki/storage/cassandra/search/NGramGeneratorTest.java

```diff
@@ -46,4 +46,16 @@ public void testNonMultipleLabelName() throws Exception {
     assertEquals("llo", ngrams.next());
     assertFalse(ngrams.hasNext());
   }
+
+  @Test
+  public void testNgramSize() throws Exception {
```

Comment:
This test is redundant since we check that `ngrams.hasNext()` is `false` for the same word in the other tests above.